### PR TITLE
fix conversion logic

### PR DESCRIPTION
**Pull request by supun-ebi**

### CODE REVIEW

File: uniprotkb-common/src/test/java/org/uniprot/api/uniprotkb/common/response/converter/UniProtKBGffMessageConverterTest.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+        uniProtKBEntryUniProtKBGffMessageConverter.before(messageContext, outputStream);
+
+        String result = outputStream.toString(StandardCharsets.UTF_8);
+        assertNotNull(result);
+        assertEquals(GFF_HEADER, result.strip());
+    }
+
+    @Test
+    void writeEntity() throws Exception {
```
Comment (LeonardoGonzales) [inline uniprotkb-common/src/test/java/org/uniprot/api/uniprotkb/common/response/converter/UniProtKBGffMessageConverterTest.java:34]: - Can you please add another test for empty, (without features)?

**Review by LeonardoGonzales — APPROVED** [review]
